src/core/Akka/Event/LoggingBus.cs

```diff
@@ -108,33 +127,23 @@ internal void StartDefaultLoggers(ActorSystemImpl system)
                     continue;
                 }
 
-                if (asyncStart)
-                {
-                    // Not awaiting for result, and not depending on current thread context
-                    Task.Run(() => AddLogger(system, loggerType, logLevel, logName, timeout))
-                        .ContinueWith(t =>
-                        {
-                            if (t.Exception != null)
-                            {
-                                Console.WriteLine($"Logger [{strLoggerType}] specified in config cannot be loaded: {t.Exception}");
-                            }
-                        });
-                }
-                else
+                AddLogger(system, loggerType, logName);
+            }
+
+            if (!Task.WaitAll(_startupState.Keys.ToArray(), timeout))
```

Comment:
We will wait for all loggers to be ready here, limiting the time to the timeout period